exercise-src/react-native/11-navigation/02-problem/src/App.tsx

```diff
@@ -12,18 +12,34 @@ import Box from "./design/Box"
 import Typography from "./design/Typography"
 import { createStackNavigator } from "@react-navigation/stack"
 
-declare global {
-  // eslint-disable-next-line @typescript-eslint/no-namespace
-  namespace ReactNavigation {
-    // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    interface RootParamList extends RestaurantsStackParamList {}
-  }
-}
 
 const RestaurantsStack = createStackNavigator()
 const RestaurantsNavigator: FC = () => {
   return (
-   //Exercise: Implement Stack Navigator that includes CityList and StateList
+    <RestaurantsStack.Navigator
+    initialRouteName="StateList"
+    screenOptions={{
```

Comment:
The indentation in this file needs to be fixed. [Be sure to look at the diffs in Bitovi Academy.]